### PR TITLE
DACO Renewal Docs Update

### PR DESCRIPTION
**Pull request by MitchellShiell**
Attached to Issue #556

### CODE REVIEW

File: docs/data-access/daco/daco-faq.md [hunk 1]
```diff
@@ -139,8 +139,8 @@ Approved users may only use the ICGC Controlled Data for the objectives and anal
...
+DACO approval will expire after two years of access. You may renew your application on the ICGC DACO website from 90 days prior to expiry and up to 90 days after expiry. For more details, see our section covering [How to Renew an Application](https://docs.icgc-argo.org/docs/data-access/daco/renew-close#how-to-renew-an-application).
 
 20. **Our access to the controlled data has expired after our 2-year approval. Can I still renew it?**
 
-If you wish to regain access, you may, within 90 days of expiry, verify or update information in your application and re-submit it for annual renewal. Beyond this period, your application will be closed and you will then have to submit a new application in order to be granted access.
+Yes, if your access has expired, you have up to 90 days post-expiry to renew your application. Beyond this 90-day window, your application will be closed, and you will need to submit a completely new application to regain access.
```
Comment (anncatton) [inline docs/data-access/daco/daco-faq.md:146]: my bad, i think i miscommunicated this part - 90 days post-expiry the application will be *permanently expired*, as opposed to closed.
The progress bar in this case would display a grey `Expired` with the `X`, instead of the red `Expired` with the `!`

I can provide screenshots for this if needed!

File: docs/data-access/daco/renew-and-close.md [hunk 2]
```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
...
+| **Expired Non-renewable** | If you do not renew the application within 90 days after expiry, the application will be closed permanently. <br/> <br/> ![DACO application in Expired Non-renewable state](/assets/data-access/daco-expired-nonrenewable.png)                                                                                    | You can download the DACO signed PDF using the **Approved PDF** top button. <br /><br /> ![Download button for DACO approved PDF](/assets/data-access/download-approved.png)                                                      |
+| **Renewal Application**   | Upon choosing to renew an application, a new draft application is generated, linked to the original application. This new draft must be completed for renewal. The original application cannot be edited. <br/> <br/> ![Renewal DACO application](/assets/data-access/daco-renewal.png)                           | A **Draft PDF** of the new application can be downloaded, which will display a watermark, useful for collaboration on the renewal process. <br /><br /> ![Download button for draft PDF](/assets/data-access/download-draft.png)  |
```
Comment (anncatton) [inline docs/data-access/daco/renew-and-close.md:17]: there are a couple editing options available on the original application, as long as it is still in `APPROVED` state - collaborators can be added/removed, and ethics docs can be added. However any changes to the original after the renewal draft is created would not be transferred to the renewal.
tldr; not sure if we should include that `The original application cannot be edited` line?

File: docs/data-access/daco/renew-and-close.md [hunk 3]
```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
...
 
 ### How to Renew an Application
 
 1. When the applicant receives the email reminder that the application is about to expire (or has expired), they can login and click the **Renew** button.
+
    ![DACO renew button for an expired application](/assets/data-access/daco-renew-button.png)
-1. At this time, the application will be unlocked and you can make any changes (if necessary) to the currently approved application, agree to the Data Access Agreement and Appendices again, and re-sign and submit the renewal application.
+
+2. Upon clicking the **Renew** button, a new, separate draft renewal application is created. This draft is linked to the original application for reference purposes. The applicant must complete this new draft application for renewal. The original application cannot be edited directly.
```
Comment (anncatton) [inline docs/data-access/daco/renew-and-close.md:44]: similar to comment above - maybe the note about editing the original application can specify that this is only available on still approved applications? or that edits to the original will not transfer to the renewal

File: docs/data-access/daco/renew-and-close.md [hunk 4]
```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
...
+
    ![DACO renewal application](/assets/data-access/daco-renewal-application.png)
-1. The DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains the same as the previous version, the project team should be quickly approved for an additional 2 years.
+
+3. The DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains consistent with the previous version, the project team should be quickly approved for an additional 2 years.
 
 ### Expired and Non-renewable
 
-A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not received within 90 days of the application's expiry date, it will be considered closed permanently.
+A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not submitted within 90 days of the application's expiry date, it will be closed permanently.
```
Comment (anncatton) [inline docs/data-access/daco/renew-and-close.md:52]: would slightly modify: `If a renewal is not submitted within 90 days of the original application's expiry date, the renewal will be closed permanently.`

File: docs/data-access/daco/renew-and-close.md [hunk 5]
```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
...
+
    ![DACO close an application button](/assets/data-access/daco-close-button.png)
-1. When clicking this button, the user will be asked to confirm this action, as it cannot be undone. If the application is being closed after it has been approved, the confirmation modal will display a warning notifying the user that this action will remove access for the project team.
-   ![DACO close confirmation modal with warning](/assets/data-access/daco-close-modal.png)
-1. Once confirming this action, the application will be closed permanently. The expiry date will change to reflect the date that the application was closed. If the applicant wishes to regain access, a new application will have to be submitted for review.
-1. The applicant and all collaborators will receive an email notifying them that the application has been closed and access to ICGC Controlled Data will be removed within 24 hours.
+
+2. When clicking this button, a confirmation window will appear, asking the user to manually enter their application ID to confirm the closure.
```
Comment (anncatton) [inline docs/data-access/daco/renew-and-close.md:62]: there are 2 steps to the confirmation:
1) the initial "are you sure, you will lose access" message -> user clicks "yes, close"
2) the appId input appears in the modal for final confirmation

Not sure if you feel it's useful to add both screenshots for this

**Review by anncatton — COMMENTED** [review]
just some minor feedback/questions!

File: docs/data-access/daco/renew-and-close.md (hunk 5 above)
Comment (MitchellShiell) [reply docs/data-access/daco/renew-and-close.md:62]: updated the text slightly, screenshot doesn't seem necessary

File: docs/data-access/daco/daco-faq.md (hunk 1 above)
Comment (MitchellShiell) [reply docs/data-access/daco/daco-faq.md:146]: photoshop will be quicker just need to change closed to expired in the image

Comment (anncatton) [reply docs/data-access/daco/daco-faq.md]: sweet! yes much easier

File: docs/data-access/daco/renew-and-close.md [hunk 6]
```diff
@@ -5,52 +5,64 @@ sidebar_label: Attesting, Renewing, & Closing
...
+
    ![DACO renew button for an expired application](/assets/data-access/daco-renew-button.png)
-1. At this time, the application will be unlocked and you can make any changes (if necessary) to the currently approved application, agree to the Data Access Agreement and Appendices again, and re-sign and submit the renewal application.
+
+2. Upon clicking the **Renew** button, a new, separate draft renewal application is created. This draft is linked to the original application for reference purposes. The applicant must complete this new draft application for renewal. The original application cannot be edited directly.
+
+3. Upon clicking the **Renew** button, a new, separate draft renewal application is created. This draft is linked to the original application for reference only. The applicant must complete this new draft application for renewal. While the original application remains in the APPROVED state some fields can be edited. Collaborators can be added or removed, and ethics documents can be appended. Any changes made do not transfer to the renewal draft.
```
Comment (anncatton) [inline docs/data-access/daco/renew-and-close.md:46]: new text looks good! i think the initial version (`2.`) got left in tho?

**Review by anncatton — COMMENTED** [review]
Looks great! I think there's just that one block of text that's a duplicate

File: docs/data-access/daco/renew-and-close.md (hunk 6 above)
Comment (MitchellShiell) [reply docs/data-access/daco/renew-and-close.md:46]: yup, that is what review is for!/just keeping you on your toes

Comment (anncatton) [reply docs/data-access/daco/renew-and-close.md:46]: lolol 💯

File: docs/data-access/daco/renew-and-close.md [hunk 7]
```diff
@@ -5,52 +5,64 @@ sidebar_label: Attesting, Renewing, & Closing
...
 
-A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not received within 90 days of the application's expiry date, it will be considered closed permanently.
+A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not submitted within 90 days of the original application's expiry date, the renewal will be permanently expired and you will need to submit a new application to regain access.
+
 ![DACO renewal period is up so application is non-renewable](/assets/data-access/daco-renewal-is-up.png)
 
 ## Closing an Application
 
-1. The application can also be closed by the Applicant or by a DACO Administrator at any time within the lifecycle of the application (before or after approval). A **Close Application** button is available from within the application header bar.
+1. The application can be closed by the Applicant or by a DACO Administrator at any time within the lifecycle of the application (before or after approval). A **Close Application** button is available from within the application header bar.
```
Comment (anncatton) [inline docs/data-access/daco/renew-and-close.md:60]: not for this PR, but just noticed the language we're using here, referring to the `Applicant` - we may need to revisit where this is used in the context of the submitter/applicant (PI) confusion

**Review by anncatton — APPROVED** [review]
Grand 🎨 - thanks @MitchellShiell !
